plugins/liip/repaircafe/models/Event.php

```diff
@@ -85,10 +85,10 @@ private function getGeocodingApiEndpoint($address)
     public function getStaticMapURL()
     {
         if (!empty($this->latitude) && !empty($this->longitude)) {
-            $api_key = Config::get('liip.repaircafe::api_key');
+            $access_tocen = Config::get('liip.repaircafe::mapbox_access_token');
```

Comment:
please write access_token :) I'm also realizing, that we maybe should have the api-key somewhere else than in github..? 😅